txnprovider/txpool/assemble.go

```diff
@@ -55,7 +56,7 @@ func Assemble(
 	}
 
 	chainID, _ := uint256.FromBig(chainConfig.ChainID)
-	maxBlobsPerBlock := chainConfig.GetMaxBlobsPerBlock()
+	maxBlobsPerBlock := chainConfig.GetMaxBlobsPerBlock(uint64(time.Now().Second()))
```

Comment:
It's better to turn `maxBlobsPerBlock` into a function that takes time – otherwise txpool will use the old value post-Pectra if it was created before the fork.